fixtures/ed/ntfs/frequencies.txt

```diff
@@ -0,0 +1,2 @@
+trip_id,start_time,end_time,headway_secs
```

Comment:
Did you launch `make docker_test` after modifying this fixture ?
It seems to me that it fails there (and probably after): https://github.com/CanalTP/navitia/blob/dev/source/ed/docker_tests/ed_integration_tests.cpp#L122 (having 10 vj instead of 8)

This fixture is linked to multiple tests... Not sure if it's worth refactor tests or just modify tests accordingly (at least it's good to test your new feature in docker_test).